LICENSE.md

```diff
@@ -1,7 +1,7 @@
-BSD 2-Clause License
+All files in this repository are copyrighted by their respective authors
```

Comment:
I think that's the formal use of the `LICENSE.md` file is to apply to the entire repo, so I'm not sure we need to state this explicitly. That said, is there any reason to deviate from the `LICENSE.md` in the `lingua-franca` repo? There, we simply state:
```
This software is licensed under the BSD 2-Clause License.

Copyright (c) 2019-2023, the Lingua Franca contributors.
All rights reserved.

(...)
```

Reply:
The copyright should be held by "authors" not "contributors" (those identified by @author tags).  For a book, for example, a copy editor does not hold the copyright.  So if it's OK with you, I will also update the LICENSE.md file in lingua-franca.  Also, "contributors" could include those who provide financial support.

Reply:
OK, that's a good point. Let me have another look at the wording before we also change the other one.

Reply:
Maybe we can just keep the default
```
BSD 2-Clause License
```
and then have something like:
```
Copyright (c) 2021-2023, the authors as specified in each individual file in this repository.
All rights reserved.
```

Reply:
But we might also just keep it simple and write
```
Copyright (c) 2021-2023, the authors.
```
because if you commit code but forget to add yourself as an author, you would still be a copyright holder.

Reply:
Just committing code does not make you an author. Again, think of a copy editor.

Also, copyright and license are two different things.  Copyright holders are the people authorized to grant a license. The LICENSE.md does not need to bear a copyright notice (and in fact, the copyright notice would only apply to the text in that file, which I don't think we can claim copyright to).

Reply:
When you select BSD-2 License on GitHub, it creates a default text that includes a Copyright notice. That's what a BSD-2 license looks like -- _it includes a copyright notice_. I agree that this is strange, but if we don't want that we need to select another license and not use BSD-2. If the BSD-2 template has a copyright notice in it, and the use of the template is for copyright owners to enter their name after the copyright statement, then there it makes no sense to interpret the copyright statement to apply to the statement itself. We didn't write that statement, so we can't copyright it.

A copyright notice also isn't necessary in order to be able to claim copyright. If I can prove that I authored something (e.g., by having sent something in a sealed envelope to myself) or checked in code to GitHub, I can claim copyright over that. I think the only utility this particular copyright statement has is to make it clear that contributors aren't signing away their right when they check code into our repository.

What problem can you identify with the statement `Copyright (c) 2021-2023, the authors`?

I recall that we have had a similar conversation a while back, and that resulted in the `LICENSE.md` that's checked into `lingua-franca`...

Reply:
Ok, let's go with that.